File: GitInsights/DataImporter/Models/Owner.cs
using Newtonsoft.Json;

namespace DataImporter.Models
{
    public class Owner
    {
        public int Id { get; set; }
        public string Login { get; set; }
        [JsonProperty("avatar_url")]
        public string AvatarUrl { get; set; }
        [JsonProperty("html_url")]
        public string HtmlUrl { get; set; }
    }
}

File: src/models/mappings/owner.py
from sqlalchemy import Column, String
from src.models.mappings.base import BaseMapping
from sqlalchemy.orm import relationship


class Owner(BaseMapping):
    login = Column(String)
    avatar_url = Column(String)
    html_url = Column(String)
    repository = relationship("Repository", back_populates="owner")

